docs/bigcommerce/installation.mdx

```diff
@@ -0,0 +1,156 @@
+
+To do so, update your `.front-commerce.js` file as below:
+
+```js title=".front-commerce.js"
+module.exports = {
+  name: "Front-Commerce Skeleton",
+  url: "http://localhost:4000",
+  modules: [
+    "./node_modules/front-commerce/theme-chocolatine",
+    // highlight-next-line
+    "./node_modules/front-commerce/modules/big-commerce",
+    "./src",
+  ],
+  serverModules: [
+    { name: "FrontCommerce", path: "server/modules/front-commerce" },
+    // highlight-next-line
+    { name: "BigCommerce", path: "big-commerce/server/modules" },
+  ],
+  webModules: [
+    { name: "FrontCommerce", path: "front-commerce/src/web" },
+    { name: "ThemeChocolatine", path: "front-commerce/theme-chocolatine/web" },
+    // highlight-start
+    {
+      name: "BigCommerce",
+      path: "front-commerce/modules/big-commerce/web",
+    },
+    {
+      name: "BigCommerceWeb",
+      path: "front-commerce/modules/big-commerce-web/web",
+    },
+    // highlight-end
+    { name: "Skeleton", path: "./src/web" },
+  ],
+};
+```
+
+## Configure the environment
+
+:::note
+
+Those environment variables are described in detail on the
+[Environments variables](/docs/reference/environment-variables#bigcommerce)
+page.
+
+:::
+
+Front-Commerce uses BigCommerce GraphQL and Management API
+
+- Management API variables: `FRONT_COMMERCE_BIG_COMMERCE_ENDPOINT` and
+  `FRONT_COMMERCE_BIG_COMMERCE_AUTH_TOKEN`
+- GraphQL variables: `FRONT_COMMERCE_BIG_COMMERCE_GRAPH_ENDPOINT` and
+  `FRONT_COMMERCE_BIG_COMMERCE_GRAPH_TOKEN`
+
+## [Access to Management API](ttps://developer.bigcommerce.com/docs/ZG9jOjIyMDYxMw-v2-and-v3-rest-api-authentication#obtaining-store-api-credentials)
+
+- In the control panel: `Advanced Settings > API Accounts > Create an API
+  Account > Create V2/V3 API Token``
+- Choose OAuth Scopes on the right
+- Save and keep the `BigCommerceAPI-credentials*.txt` file
+
+In your `.env` file, define:
+
+- `FRONT_COMMERCE_BIG_COMMERCE_ENDPOINT` with `API PATH` value in the text file
+  (without the `/v3/` suffix)
+- `FRONT_COMMERCE_BIG_COMMERCE_AUTH_TOKEN` with `ACCESS TOKEN` value in the text
+  file
+
+From that `.txt` file, also fill:
+
+- `FRONT_COMMERCE_BIG_COMMERCE_CLIENT_ID` with `CLIENT ID`
+- `FRONT_COMMERCE_BIG_COMMERCE_CLIENT_SECRET` with `CLIENT SECRET`
+- `FRONT_COMMERCE_BIG_COMMERCE_STORE_HASH` with the store hash
+
+Example:
+
+```shell title=".env"
+FRONT_COMMERCE_BIG_COMMERCE_ENDPOINT=https://api.bigcommerce.com/stores/yhowbpps2e
+FRONT_COMMERCE_BIG_COMMERCE_AUTH_TOKEN=ab9cd9x75abc1tumu0wce8yxedu3ue3
+FRONT_COMMERCE_BIG_COMMERCE_CLIENT_ID=aclientid
+FRONT_COMMERCE_BIG_COMMERCE_CLIENT_SECRET=aclientsecret
+FRONT_COMMERCE_BIG_COMMERCE_STORE_HASH=yhowbpps4e
+```
+
+## [Access to BigCommerce Graph](https://developer.bigcommerce.com/docs/ZG9jOjIyMDczOQ-overview)
+
+[Create a _Customer impersonation tokens_](https://developer.bigcommerce.com/docs/ZG9jOjIyMDczOQ-overview#customer-impersonation-tokens)
```

Comment:
```suggestion
[Create a _Customer impersonation token_](https://developer.bigcommerce.com/docs/ZG9jOjIyMDczOQ-overview#customer-impersonation-tokens)
```

Reply:
**nit:** I think making a link here is a bit confusing. Links in the docs are red, so is the header just above (it looks like a subtitle).

![image](https://user-images.githubusercontent.com/4162177/194560869-57fb27fc-e589-4386-b48e-db5108654f5a.png)


**suggestion:**
See: [Create a _Customer impersonation token_](https://developer.bigcommerce.com/docs/ZG9jOjIyMDczOQ-overview#customer-impersonation-tokens)

Reply:
Fixed in aa0e7e4